src/sources/youtube.js

```diff
@@ -518,22 +512,17 @@ async function loadFrom(query, type) {
     }
     case constants.YouTube.shorts: {
       debugLog('loadtracks', 4, { type: 1, loadType: 'track', sourceName: 'YouTube Shorts', query })
-
+      
       const { body: short } = await makeRequest(`https://${_getBaseHostRequest(type)}/youtubei/v1/player`, {
-        headers: {
-          'User-Agent': ytContext.client.userAgent,
-          'X-GOOG-API-FORMAT-VERSION': '2',
-          ..._additionalHeaders
-        },
-        body: {
+        body: {  
           context: ytContext,
           videoId: /shorts\/([a-zA-Z0-9_-]+)/.exec(query)[1],
           contentCheckOk: true,
-          racyCheckOk: true,
-          params: 'CgIQBg'
+          racyCheckOk: true
         },
-        method: 'POST'
-      })
+        method: 'POST',
+        disableBodyCompression: true
+      });
```

Comment:
```suggestion
      })
```


Do not add semicolons